exporters/jaeger/build.gradle

```diff
@@ -0,0 +1,63 @@
+description = 'OpenTelemetry - Jaeger Exporter'
+
+apply plugin: 'com.google.protobuf'
+
+def protobufVersion = '3.7.1'
+def protocVersion = '3.7.1'
+
+buildscript {
+    repositories {
+        maven { url "https://plugins.gradle.org/m2/" }
+    }
+    dependencies {
+        classpath "com.google.protobuf:protobuf-gradle-plugin:0.8.8"
+    }
+}
+
+dependencies {
+    api project(':opentelemetry-api')
+    api project(':opentelemetry-sdk')
```

Comment:
Should be SDK as provided dependency?

Reply:
Did you mean `implementation` instead of `provided`? I think it's appropriate to have `api` here, as methods from this module accept and return instances of classes defined by the SDK module.

> An API dependency is one that contains at least one type that is exposed in the library binary interface, often referred to as its ABI (Application Binary Interface).

https://docs.gradle.org/current/userguide/java_library_plugin.html#sec:java_library_recognizing_dependencies

Reply:
How is `api` translated to maven dependency type? Is it `compile` or `provided`? I thought it is `provided`.

> Provided means that you need the JAR for compiling, but at run time there is already a JAR provided by the environment so you don't need it packaged with your app. For a web app, this means that the JAR file will not be placed into the WEB-INF/lib directory.

Reply:
The differences in Gradle are mostly about what is exported to consumers of the library: an application declaring a dependency on the `exporters/jaeger` module will have `opentelemetry-sdk` as a transitive dependency, as it's declared as `api`. The dependencies that `exporters/jaeger` declare as `implementation` aren't transitive dependencies for a consumer.

There's a `runtimeOnly` mode as well, which is probably close to what `provided` does in Maven. But still, I don't think it's the right mode. `api` still seems the most appropriate here.